tests/src/Interop/CMakeLists.txt

```diff
@@ -1,4 +1,12 @@
 
+if(WIN32)
+	if(CLR_CMAKE_HOST_ARCH STREQUAL arm)
+	    list(APPEND LINK_LIBRARIES_ADDITIONAL
```

Comment:
No, it is a custom variable. I believe CMake will define it if it is not already defined.